receptorctl/receptorctl/cli.py

```diff
@@ -31,14 +31,28 @@ def parse_args(self, ctx, args):
 @click.pass_context
 @click.option('--socket', envvar='RECEPTORCTL_SOCKET', required=True, show_envvar=True,
               help="Control socket address to connect to Receptor (defaults to Unix socket, use tcp:// for TCP socket)")
-def cli(ctx, socket):
+@click.option('--yaml', default=None, envvar='RECEPTORCTL_YAML', required=False, show_envvar=True,
```

Comment:
I think this should be named `--config` for consistency with the main receptor CLI.

Reply:
fixed, can now do `--config` or `-c`